packages/react/src/themes/teams/categoryColors.ts

```diff
@@ -466,7 +466,7 @@ export const categoryColors: TeamsCategoryColors = {
     350: '#C1CACC',
     400: '#ACABBB',
     450: '#A9B5B9',
-    500: '#A2A0B2',
+    500: '#A0AEB2',
```

Comment:
Please add a changelog entry, so it would be easy to track when this was changed:

> - Updated steel light 500 variant in Teams theme @natashamayurshah ([#2089](https://github.com/microsoft/fluent-ui-react/pull/2089))

Reply:
Done.